now.json

```diff
@@ -1,12 +0,0 @@
-{
```

Comment:
preco si toto vyhodil ? to sa pouziva ked chcem publishnut demo.

vratit a upravit ak treba

Reply:
Je spat (schvalne som nevyhadzoval...robil som totiz full novy build vsetkeho ked som sa mrvil este s tym p11n balikom... :( )